content/en/docs/architecture/ci-operator-internals/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "CI Operator Internals"
+description: A detailed view of some aspects of the ci-operator implementation, intended for developers working on its source code.
+---
+
+This section describes in detail some aspects of the implementation of
+`ci-operator` and associated programs, with the intent of serving as an
```

Comment:
`associated programs`: Is it about the existing content of this page, or something you plan to add?